manifests/server.pp

```diff
@@ -9,6 +9,7 @@
   Boolean $escape_ldapi_ifs,
   Array[String[1]] $ldapi_ifs,
   Stdlib::Absolutepath $default_directory,
+  Boolean $manage_repo = true,
```

Comment:
should we rename it to `include_epel` or `manage_epel` maybe? there are also openldap repos out there, maybe the param will confuse people and they assume they will get upstream repos

Reply:
Renamed to `manage_epel`